### ISSUE TITLE
[BUG] On "KNOW" the list of values in the "Get Related" menu is not always correct

### ISSUE THREAD

**Issue by sjordan1975** [marklogic-community/envision#144]
*### Welcome Envision enthusiast!

Let's get the most out of this ticket, please fill out this template.

#### The issue

Short description of the problem:

When the ‘Know’ page loads, it seems to scan for all predicates on the page and create an array for use in the menu of ‘Get Related’. This results in the menu being the same regardless of the node on the page.

What behavior are you expecting?

Each node should be evaluated in its own context and the appropriate menu created

What are the steps to reproduce the issue:

#### Tech details

Which Operating System are you using? OSX

Which version of MarkLogic are you using? 10.0.5

Which version of the Data Hub are you using? 5.2.4

#### The devil is in the details...

See attached [KNOW_page_load_response.txt](https://github.com/marklogic-community/envision/files/5721856/KNOW_page_load_response.txt)

**Comment by paxtonhare** [comment]
Hey @sjordan1975 , I'm looking into this one. To clarify, are you saying that the "/Pest/DAWR/Assessment/Location/0a5ab0ff-1534-4085-8a5e-82450d6eca1c-2011-07-01T00:00:00.000Z.json" node right click menu has more predicates than reality? Or are you saying that other nodes like "/Taxonomy/ID-f27d7d86-1b2f-4352-bd9c-73820ea0755c.json" have more predicates on the menu?

I'm trying to determine if the UI is showing predicates on the wrong nodes or if the backend sjs service is adding extra predicates to the list for each node.

**Comment by sjordan1975** [comment]
Hey @paxtonhare sorry about the delay.

I have attached 2 screenshots which I hope help explain.

In Screen Shot 1 I have searched for the string "ID-f27d7d86-1b2f-4352-bd9c-73820ea0755c".

KNOW has returned a Taxonomy Subject and an Assessment Subject (see annotations).

In Screenshot 2 I have right-clicked on the Assessment. As an example, the predicate "hasCommonName" is showing as related to the Assessment (see annotation).

I may misunderstand how Envision works, but I'd assume "hasCommonName" should not be on the Assessment as that is not a predicate for the Assessment subject; rather, it's a predicate of a Taxonomy.

<img width="1400" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/2862727/103498143-d675ba80-4e97-11eb-8ab8-54987f5acf56.png">

<img width="1402" alt="Screen Shot 2" src="https://user-images.githubusercontent.com/2862727/103498159-dfff2280-4e97-11eb-8685-54909448a3ba.png">
